README.md

```diff
@@ -34,7 +47,57 @@
 
 ## Developers' Notes
 
-### **I. Database SCHEMA**
+## **I. Technologies Implemented**
+
++ PostgreSQL 12.1
++ Express.js 4.16.1
++ React 16.12.0
++ React/Redux 7.2.0
++ Redux 4.0.5
++ React Router Web 5.1.2
++ Pg-promise 10.4.4
++ Axios 0.19.2
++ Bootstrap 4.4.1
++ React Bootstrap Carousel 4.0.3
++ Multer 1.4.2
++ Node.js 13.8.0
++ JavaScript 2019 / ECMAScript 10
++ HTML5
++ CSS3
++ Git / Github
+
+## **II. Setup**
+
++ To run this project [back end], install it locally using npm:
+  ```
+  $ npm install
+  $ npm start
+  ```
++ Instantiate and seed the PostgresQL database:
+  ```
+  $ cd database
+  $ psql -f greenlist_registry_db.sql
+  ```
++ To run the full stack of Greenlist, install the [front end](https://github.com/joseph-p-pasaoa/bingebook_frontE__FullStack) locally afterwards, as well, in a separate folder using npm:
```

Comment:
It’s the wrong URL. GreenList not bingebook.

Reply:
@KathyPuma